# HG changeset patch
# User Rik <rik@octave.org>
# Date 1551662432 28800
#      Sun Mar 03 17:20:32 2019 -0800
# Branch stable
# Node ID e07ea25d58f56b359fdc41eec99a22a9fb6b3b2c
# Parent  677764865056ce97f892c0c6a011e7772749823f
quiver3.m: Fix texinfo typo in docstring from cset f7b10bd40045.

* quiver3.m: Put @deftypefn entry first in docstring.

diff --git a/scripts/plot/draw/quiver3.m b/scripts/plot/draw/quiver3.m
--- a/scripts/plot/draw/quiver3.m
+++ b/scripts/plot/draw/quiver3.m
@@ -12,18 +12,18 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <https://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefnx {} {} quiver3 (@var{x}, @var{y}, @var{z}, @var{u}, @var{v}, @var{w})
-## @deftypefn  {} {} quiver3 (@var{z}, @var{u}, @var{v}, @var{w})
+## @deftypefn  {} {} quiver3 (@var{x}, @var{y}, @var{z}, @var{u}, @var{v}, @var{w})
+## @deftypefnx {} {} quiver3 (@var{z}, @var{u}, @var{v}, @var{w})
 ## @deftypefnx {} {} quiver3 (@dots{}, @var{s})
 ## @deftypefnx {} {} quiver3 (@dots{}, @var{style})
 ## @deftypefnx {} {} quiver3 (@dots{}, "filled")
 ## @deftypefnx {} {} quiver3 (@var{hax}, @dots{})
 ## @deftypefnx {} {@var{h} =} quiver3 (@dots{})
 ##
 ## Plot a 3-D vector field with arrows.
 ##
